packages/components/src/List/Toolbar/ColumnChooserButton/ColumnChooser/ColumnChooserHeader/ColumnChooserHeader.component.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useColumnChooserContext } from '../columnChooser.context';
+import Tooltip from '../../../../../Tooltip';
+import cssModule from '../ColumnChooser.scss';
+import { getTheme } from '../../../../../theme';
+
+const theme = getTheme(cssModule);
+
+const filterVisibleColumns = column => column.visible;
+
+const Default = () => {
+	const { columnsChooser, t } = useColumnChooserContext();
+	const selectedColumns = t('SELECT_COLUMNS', {
+		defaultValue: 'selected columns',
+	});
+	const lengthVisibleColumns = columnsChooser.filter(filterVisibleColumns).length;
+	return (
+		<div>
+			<div className={theme('tc-column-chooser-header-title')}>
+				{t('COLUMN_CHOOSER_HEADER_TITLE', {
+					defaultValue: 'Modify columns',
+				})}
+			</div>
+			<div id="selected-columns-text">{`${lengthVisibleColumns}/${columnsChooser.length} ${selectedColumns}`}</div>
```

Comment:
Include the numbers in the translation. The `lengthVisibleColumns ` must be named `count` in translation options, so i18n-scanner will generate 2 keys: singular and plural.
The default value must be singular BTW